gateway/1.5/modules/ROOT/pages/policies-pdk-develop-custom-policies.adoc

```diff
@@ -2,4 +2,48 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+After installing the xref:policies-pdk-prerequisites.adoc[], you can develop custom policies by using the Policy Development Kit (PDK).
+
+The custom policy lifecycle consists of the following docs that explain all steps of developing a custom policy that occur in PDK. The following docs do not include steps that occur outside of PDK, such as xref:policies-pdk-deploy-policies.adoc[applying a policy].
```

Comment:
I would refer to steps here and not docs. 
occur > occurs
Text is confusing - does the policy occur in PDF or is PDK used to define it?
Is there a place where all of the steps are listed, whether or not they are developed in the PDK?

The custom policy lifecycle consists of the following steps for developing a custom policy by using PDF. For a list of all steps, including those that occur outside of PDK, see xxx.